examples/parcel/src/ui/typescript.tsx

```diff
@@ -8,7 +9,7 @@ interface TypeScriptProps {
 export const TypeScript = ({ color }: TypeScriptProps): JSX.Element => (
   <div
     css={{
-      display: 'inline-block',
+      display2: 'inline-block',
```

Comment:
display2?

Reply:
oops forgot to remove this when testing the types